GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+
+# Project Governance
+
+<!-- TOC -->
+
+* [Collaborators](#collaborators)
+  * [Collaborator activities](#collaborator-activities)
+  * [Collaborator nominations](#collaborator-nominations)
+* [Technical steering committee](#technical-steering-committee)
+  * [TSC meetings](#tsc-meetings)
+  * [Consensus seeking process](#consensus-seeking-process)
+* [Release Manager](#release-manager)
+
+<!-- /TOC -->
+
+
+## Collaborators
+
+Core collaborators for Systelab libraries are responsible for the GitHub repository maintenance. 
+All members of @systelab/jee-team are potential collaborators to this library.
+
+Collaborators have:
+
+* Commit access to the repository in any branch different from master.
+* Access to the Continuous integration (CI) jobs
+
+Both collaborators and non-collaborators may propose changes to the library source code. 
+The mechanism to propose such an implementation change is a GitHub pull request.
+
+Collaborators review and merge (_land_) pull requests.
+
+Three collaborators must approve a pull request before the pull request can land. 
+Approving a pull request indicates that the collaborator accepts
+responsibility for the change. Approval must be from collaborators who are not
+authors of the change.
+
+If a collaborator opposes a proposed change, then the change cannot land. Often, discussions or further changes
+result in collaborators removing their opposition.
+
+Please see the detailed instructions on [Contribution](https://github.com/systelab/java-report-generator/blob/master/CONTRIBUTING.md)
+
+### Collaborator activities
+
+* Helping users and novice contributors
+* Contributing code and documentation changes that improve the project
+* Reviewing and commenting on issues and pull requests
+* Participation in working groups
+* Merging pull requests
+
+The TSC can add and remove collaborators to the @systelab/jee-team team.
+
+### Collaborator nominations
+
+Existing collaborators can nominate someone to become a collaborator. Nominees
+should have significant and valuable contributions across the Sytelab libraries
+organization.
+
+To nominate a new collaborator, contact someone with GitHub account Administrator rights in order to add the new collaborator to the systelab/jee-team.
+
+The TSC can remove collaborators from the @systelab/jee-team team by contacting someone with GitHub account Administrator rights.
+
+## Technical Steering Committee
+
+A subset of the collaborators forms the Technical Steering Committee (TSC).
+The TSC has final authority over this project, including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of collaborators
+
+The current list of TSC members for this library is:
+
+- Olga Puig @olgapuig (Meeting Chair)
```

Comment:
I suggest to include Oscar Galceran and Quim Noto as the meeting chairs for the Java TF